src/lib/db/index.ts

```diff
@@ -200,6 +244,14 @@ export default class Database {
       .where({ 'demand.id': id, subtype })
   }
 
+  insertDemandComment = async (comment: object) => {
+    return this.db().demand_comment().insert(comment).returning('*')
+  }
+
+  updateDemandComment = async (id: UUID, comment: object) => {
+    return this.db().demand_comment().update(comment).where({ id }).returning('*')
```

Comment:
missing `updated_at: this.client.fn.now()`

Reply:
yes